docs/minio-client-complete-guide.md

```diff
@@ -279,7 +234,7 @@ USAGE:
    mc ls [FLAGS] TARGET [TARGET ...]
 
 FLAGS:
-  --help, -h					Help of ls.
+  --help, -h                       Show help.
```

Comment:
Aligned explanations may look nicer for this and following `FLAGS;` and `OPTIONS:` sections.

Reply:
Done